halo2_proofs/benches/fft.rs

```diff
@@ -16,7 +17,7 @@ fn criterion_benchmark(c: &mut Criterion) {
             let mut a = (0..(1 << k)).map(|_| Fp::random(OsRng)).collect::<Vec<_>>();
             let omega = Fp::random(OsRng); // would be weird if this mattered
             b.iter(|| {
-                best_fft(&mut a, omega, k as u32);
+                best_fft(&mut a, omega, k as u32, &FFTData::default(), false);
```

Comment:
I think this will fail when running this with the recursive FFT because `FFTData::default()` won't contain the valid data. However this is just a performance benchmark so ehhh if you can quickly generate the right data here (I think not hard) please do so, otherwise also ok to just leave it as is.

Reply:
Fixed in latest.